Source/OxyPlot/PlotController/Manipulators/PlotManipulator.cs

```diff
@@ -77,6 +84,21 @@ protected void AssignAxes(ScreenPoint position)
             if (this.PlotView.ActualModel != null)
             {
                 this.PlotView.ActualModel.GetAxesFromPoint(position, out xaxis, out yaxis);
+
+                if (AxisPreference != AxisPreference.None &&
```

Comment:
Member access should use `this`: if you can write a style-cop pass that would be great.

Reply:
I added a rule in .editorconfig for this.